derive_builder_test/Cargo.toml

```diff
@@ -9,7 +9,7 @@ publish = false
 path = "src/lib.rs"
 
 [features]
-nightlytests = ["compiletest_rs", "derive_builder/nightlytests"]
+nightlytests = ["compiletest_rs", "derive_builder/nightlytests", "derive_builder/try_from"]
```

Comment:
I removed the `derive_builder_test` crate in latest master. :-)

PS: More correctly - I merged it into the `derive_builder` crate and left a tiny `derive_builder/tests/dummy-crate` for doc-inspection.